examConfig.js

```diff
@@ -606,6 +606,14 @@ export const josaaConfig = {
       label: "Select Your Home State",
       options: statesList,
     },
+    {
+      name: "preferHomeState",
+      label: "Would you like to prioritize colleges in your home state?",
```

Comment:
I feel like this wording could be simplified?

"Do you want us to list colleges in {{homeState}} first?"

You can replace the value of {{homeState}} with what they selected above?